### DISCUSSION TITLE
Running wam2layers with ERA5

### DISCUSSION THREAD

**Discussion by karelsanchez** [WAM2layers/WAM2layers#370] [Q&A]
Dear academic researcher group, I'm new in this forum I would like to know if it's possible to run wam2layers using ERA5 data?, how should the data be to be able to run it? Single file in NC or how? I appreciate your support.

**Comment by ruudvdent** [comment]
Dear @karelsanchez, welcome to WAM2layers. Please introduce yourself and your research. We're working on extending the example data, but if you follow the [user guide](https://wam2layers.readthedocs.io/en/latest/userguide.html), you should have all answers on downloading ERA5 data and running the preprocessing and tracking. Let us know if you have more specific questions and/or when you're succesful!
